### PR TITLE
Added note about p2p host

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added note about p2p host

Waiting on ELK additions to Quorum Dev Quickstart to resolve broken links.

### CODE REVIEW

File: docs/Concepts/Permissioning/Onchain-Permissioning.md [hunk 1]
```diff
@@ -56,6 +56,15 @@ Permissioning implements three allowlists:
     If using account permissioning and privacy, a signing key must be specified using the
     [`--privacy-marker-transaction-signing-key-file`] command line option and the signing key
     included in the accounts allowlist.
+    
+!!! tip 
+  
+    If nodes are not connecting as expected, set the [log level to `TRACE`](../../Reference/CLI/CLI-Syntax.md#logging)
+    to identify the issue.
```
Comment (lucassaldanha) [inline docs/Concepts/Permissioning/Onchain-Permissioning.md:63]: Probably worth adding a bit more info on how to find problems. Something like:


> If nodes are not connecting as expected, set the [log level to `TRACE`](../../Reference/CLI/CLI-Syntax.md#logging) and search for messages containing `Node permissioning` to identify the issue.

File: docs/Concepts/Permissioning/Onchain-Permissioning.md [hunk 2]
```diff
@@ -56,6 +56,15 @@ Permissioning implements three allowlists:
...
     included in the accounts allowlist.
+    
+!!! tip 
+  
+    If nodes are not connecting as expected, set the [log level to `TRACE`](../../Reference/CLI/CLI-Syntax.md#logging)
+    to identify the issue. 
+    
+    Ensure the [`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host) command line option has been
+    correctly configured for all nodes with the 
+    externally accessible address.  If not specified, nodes advertise `0.0.0.0`.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Concepts/Permissioning/Onchain-Permissioning.md:68]: Removed

File: docs/Concepts/Permissioning/Onchain-Permissioning.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Concepts/Permissioning/Onchain-Permissioning.md]: Updated

**Review by bgravenorst — APPROVED** [review]
Looks good to me

**Pull request merged**